discord/commands/count.js

```diff
@@ -0,0 +1,168 @@
+/** This is a test command to show what command files should look like */
```

Comment:
we should probably clean up these comments, but it doesn't matter much either way

Reply:
how obvious is it that i'm just copying and pasting ping whenever i make a new command